src/docs/deployment/android.md

```diff
@@ -87,8 +88,9 @@ keytool -genkey -v -keystore c:/Users/USER_NAME/key.jks -storetype JKS -keyalg R
   'Java binary at:'. Then use that fully qualified path
   replacing `java` (at the end) with `keytool`.
   If your path includes space-separated names,
-  such as `Program Files`, place quotes around the
-  space-separated names. For example: `/"Program Files"/`
+  such as `Program Files`, use a delimeter appropriate
```

Comment:
delimeter -> delimiter... but I'm not sure _delimiter_ is the word we want here. Looking up the definition of delimiter, it's a character that separates parts of a string. Maybe:

...use platform-appropriate notation for the names. For example, on Mac/Linux use ..., and on Windows use ....